edi_webservice_oca/models/edi_exchange_record.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2022 Foodles (http://www.foodles.co).
+# @author Pierre Verkest <pierreverkest84@gmail.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class EDIExchangeRecord(models.Model):
+    """
+    Define an exchange record.
+    """
+
+    _name = "edi.exchange.record"
+    _inherit = [
+        "edi.exchange.record",
+        "webservice.consumer.mixin",
+    ]
+
+    def _compute_ws_response_content_filename(self):
```

Comment:
I'm not sure this override is needed.... the file will always be attached to the current record and if you include the datetime (as mentiond in https://github.com/OCA/web-api/pull/22) we probably don't need this change.

Reply:
I wanted to reuse efficiently all the business rules implemented here https://github.com/OCA/edi/blob/14.0/edi_oca/models/edi_exchange_type.py#L213

Reply:
ok, but the file name shouldn't be the same, especially if we include the datetime as mentiond on OCA/web-api#22

Reply:
ok you are right ! I'll improve that